apps/ledger-live-desktop/src/renderer/screens/settings/sections/Developer/FeatureFlagsSettings/index.tsx

```diff
@@ -56,7 +56,7 @@ export const FeatureFlagContent = withV3StyleProvider((props: { expanded?: boole
   }, [featureFlags, searchInput]);
 
   const filteredGroups = useMemo(() => {
-    return Object.keys(groupedFeatures)
+    return (Object.keys(groupedFeatures) as Array<keyof typeof groupedFeatures>)
```

Comment:
A bit strange that we need to use the as keyword here, `Object.keys` is suppose to be typed correctly with typescript by default

Reply:
Actually `Object.keys` returns an array of string, I can create a typeguard to avoid using `as`

Reply:
You need to export the type from the library to have the correct type:
`export type GroupedFeature = "europa" | "disableNft";`
<img width="689" alt="image" src="https://github.com/LedgerHQ/ledger-live/assets/31533861/628ce4fe-ce8c-4985-9995-293fc6a32765">